Makefile

```diff
@@ -171,6 +170,24 @@ push-tenant-manager-image:
 	@echo "------------------------------------------------------------------------"
 	docker push $(IMAGE_REPO)/apicurio/apicurio-registry-tenant-manager-api:$(IMAGE_TAG)
 
+.PHONY: build-tenant-manager-image-native ## Builds native docker image for tenant manager. Variables available for override [IMAGE_REPO, IMAGE_TAG]
+build-tenant-manager-image-native:
+	@echo "------------------------------------------------------------------------"
+	@echo " Building Native Image For Tenant Manager"
+	@echo " Repository: $(IMAGE_REPO)"
+	@echo " Tag: $(IMAGE_TAG)-native"
+	@echo "------------------------------------------------------------------------"
+	docker build -f multitenancy/tenant-manager-api/src/main/docker/Dockerfile.native -t $(IMAGE_REPO)/apicurio/apicurio-registry-tenant-manager-api:$(IMAGE_TAG)-native ./multitenancy/tenant-manager-api/
```

Comment:
I have done the exact same changes, except that I have used another repository `apicurio/apicurio-registry-tenant-manager-api-native` dedicated for tenant manager native image, just like we have for native sql and kafkasal images